core-plugins/src/test/java/io/cdap/plugin/transform/StructuredtoAvroTest.java

```diff
@@ -97,7 +97,6 @@ public void testOutputSchemaUsage() throws Exception {
     GenericRecord result = avroTransformer.transform(record);
     Assert.assertEquals(123L, result.get("id"));
     Assert.assertEquals("ABC", result.get("name"));
-    Assert.assertNull(result.get("age"));
```

Comment:
with the change in avro version, it actually results in an exception being thrown instead of a null when you try to get a field that isn't in the schema.

Can you think of any problems with this? This is incompatible behavior in general, but I don't think it should be a problem in the formats.

Reply:
👍  Don't think there will be a problem as long as the schema set is correct